tests/test_rules.py

```diff
@@ -180,9 +180,9 @@ async def test_generate_rules():
     print(durable_rulesets[0][0].define())
     assert_fact("Demo rules", {"payload": {"text": "hello"}})
 
-    assert ruleset_queue_plans[0][2].get_nowait()[1] == "slack"
-    assert ruleset_queue_plans[0][2].get_nowait()[1] == "assert_fact"
-    assert ruleset_queue_plans[0][2].get_nowait()[1] == "log"
+    assert ruleset_queue_plans[0][2].get_nowait().action == "slack"
```

Comment:
@hsong-rh Can one of these tests be modified to check the ruleset name and the rule name?